CODEOWNERS

```
@@ -48,3 +49,19 @@ utils/writeable/                  @sffc @robertbastian
 utils/yoke/                       @Manishearth @sffc
 utils/zerofrom                    @Manishearth @sffc
 utils/zerovec/                    @Manishearth @sffc
+
+components/calendar/data             @robertbastian
```

should we just have this control the data.json (so that you're not a required review every time a crate tweaks its data format)

---

Is it possible to set no-owner? I don't want crate owners to be pulled in when I change the baked format.

---

Doesn't seem to be possible

---

Should we use `@unicode-org/icu4x-owners` for this? Does it have the effect we want?

---

I'm willing to let this stay as Robert rather than continuing to propagate the problem where PRs get a million reviewers.

icu4x-owners is nice though, since most PRs _already_ have that. It's the equivalent of no-owner for us.

---

I'll try icu4x-owners